latest/tests/test_validation.py

```diff
@@ -1,73 +1,85 @@
 import json
 import glob
 
+from dataclasses import dataclass
+from typing import List
+
 import pytest
 
-from jsonschema import RefResolver, Draft202012Validator
+from jsonschema import RefResolver, Draft7Validator
```

Comment:
I can't remember why we used the 202012 implementation. Is the goal here to have use the common-denominator? In that case would the version-agnostic `Validator` API be an option?

Reply:
My local version no longer had the dated Draft so I went with the latest. @will-moore may be able to say more.

Reply:
Sorry, don't remember